docs/src/main/mdoc/datatypes/freemonad.md

```diff
@@ -133,6 +133,8 @@ def update[T](key: String, f: T => T): KVStore[Unit] =
   } yield ()
 ```
 
+If the words in the grammar do not take type parameters, a shorthand is available after importing `cats.free.implicits._`: one can just write `Delete.free("key")` without defining `def delete(key: String)` first.
```

Comment:
can you extend the example here? I still don't totally follow the limitations and the benefits. It might be nice to be explicit with an example showing the verbose way, and the way using the syntax.